File: official/docs/csharp/batches/RemoveShipmentFromBatch.cs
using EasyPost;

EasyPost.ClientManager.SetCurrent("EASYPOST_API_KEY");

Batch batch = await Batch.Retrieve("batch_...");

batch.RemoveShipments("shipments", new List<Dictionary<string, object>>(){
    new Dictionary<string, object>(){
        { "id", "shp_..."}
     }
    }
 );

Console.Write(batch);

File: official/docs/go/batches/remove_shipment_from_batch.go
package example

import (
    "os"

    "github.com/EasyPost/easypost-go/v2"
)

func main() {
	apiKey := os.Getenv("EASYPOST_API_KEY")
    client := easypost.New(apiKey)

    shipment, err := client.GetShipment("shp_...")

    batch, err := client.RemoveShipmentsFromBatch("batch_...", shipment)

	fmt.Println(batch) 
}
